src/js/index.js

```diff
@@ -83,6 +95,20 @@ function cascade(force = false) {
 	if (force || this === sortingInput) {
 		handleSort(this === sortingInput);
 	}
+
+	if (this === showExpired) {
+		if (!search) {
+			search = new URLSearchParams(window.location.search);
```

Comment:
Is this necessary for this case only (`handleExpired`)?

Reply:
Not really, but I'm not sure if this is even necessary at all. I put it there to be safe, but I guess we could just move it to `cascade`.